app/page.tsx

```diff
@@ -1,7 +1,34 @@
-export default function Index(){
-    return(
-        <h1>
-            Stellar Wallet
-        </h1>
-    )
-}
\ No newline at end of file
+export default function Index() {
+  return (
+    <div className="grid grid-cols-1 grid-rows-6 h-screen bg-cyan-950">
+        
+      {/* Brand */}
```

Comment:
Remember that it's not necessary to add comments to the code.

Reply:
Thank you, I have removed these comments and commited the changes.